duckduckgo/Android
# Autocomplete

Reviewer: @subsymbolic 
Asana: https://app.asana.com/0/0/364367284740785
and also fix: https://app.asana.com/0/0/372048568323481

**Description**:  Implement autocomplete


**Steps to test this PR**:
1. type text into the Omnibar
2. confirm that autocomplete results show below the Omnibar
3. confirm that pressing the result will launch the search with the results or website and clicking the "+" icon add the suggestion on the Omnibar
4. confirm that clearing text, and pressing back both the back button and the toolbar back button dismiss the autocomplete results

###### Reviewer Checklist:


###### PR DRI Checklist:
